sdk/hdinsight/Azure.ResourceManager.HDInsight/src/Customize/HDInsightClusterGatewaySettings.Serialization.cs

```diff
@@ -26,6 +26,11 @@ internal static HDInsightClusterGatewaySettings DeserializeHDInsightClusterGatew
                         property.ThrowNonNullablePropertyIsNull();
                         continue;
                     }
+                    if (property.Value.ValueKind == JsonValueKind.String)
```

Comment:
Is this always string value?

Reply:
yes, [API](https://docs.microsoft.com/en-us/rest/api/hdinsight/2021-06-01/clusters/get-gateway-settings?tabs=HTTP#gatewaysettings) always returns the string value "true" or "false"